### PR TITLE
remove USE_SQL_BACKEND flag

### PR DESCRIPTION
**Pull request by snopoke**
@calellowitz 

I've verified that all domains have been migrated:
- https://www.commcarehq.org/hq/flags/edit/sql_backend/
- https://india.commcarehq.org/hq/flags/edit/sql_backend/

### CODE REVIEW

**Comment by snopoke** [comment]
FYI @dimagi/scale-team

File: corehq/form_processor/utils/general.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from django.conf import settings
 
-from corehq.toggles import USE_SQL_BACKEND, NAMESPACE_DOMAIN, NEW_EXPORTS, TF_USES_SQLITE_BACKEND
+from corehq.toggles import NAMESPACE_DOMAIN, NEW_EXPORTS, TF_USES_SQLITE_BACKEND
```
Comment (dimagimon) [inline corehq/form_processor/utils/general.py]: F401 'NAMESPACE_DOMAIN' imported but unused

**Comment by czue** [comment]
hooray!

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Pull request merged**